DevelopGitHub/GitHubExtension/GitHubExtension.Templates/ExtensionMethods/HttpRequestMessageExtension.cs

```diff
@@ -19,14 +23,48 @@ public static class HttpRequestMessageExtension
             }
         };
 
-        public static HttpRequestMessage CreateMessage(this HttpRequestMessage message)
+        public static HttpRequestMessage
+        CreateTemplateMessage(this HttpRequestMessage requestMessage, string gitHubMessage, string content)
+        {
+            requestMessage.AddHeadersToMessage();
```

Comment:
But this extension is nice because it works only with HttpRequestMessage and that's all.